src/components/SearchList.jsx

```diff
@@ -0,0 +1,38 @@
+import { useHistory, useParams } from 'react-router';
+
+import Link from './Link';
+import useAPIRequest from './CustomHooks/useAPIRequest';
+import { REQUEST_TYPES } from '../common/constants';
+
+const SearchList = () => {
+  const { location } = useParams();
+  const history = useHistory();
+  const locations = useAPIRequest(REQUEST_TYPES.location, location);
+
+  const handleSelectCity = e => {
+    const selectedCityId = e.target.dataset.target;
+    const selectedCityName = e.target.dataset.name;
+
+    history.push(`/home?city=${selectedCityName}&id=${selectedCityId}`);
+  };
+
+  return (
+    <div className="app__main">
+      <h2>Here's what we found:</h2>
+      {locations.length > 0 ? (
```

Comment:
At the initial point will the user see something similat to this?
```
<h2>Here's what we found:</h2>
<p>Ooops, looks like we couldn't find anything</p>
```

1) too many words find/found too close to each other.
2) the user will see this text unitl something is actually loaded

If that is the case I'd suggest to work with UX a bit more.

Reply:
I fixed that when I connected redux & added isLoading. Is it ok to keep it that way for now?

Reply:
sure